Dockerfile

```diff
@@ -6,15 +6,10 @@ COPY ./ /usr/local/app/
 
 RUN npm install
 
-RUN npm run build
+RUN npm run build --configuration=production
 
 FROM nginx:latest
 
 COPY --from=build /usr/local/app/dist/gdi-userportal-app /usr/share/nginx/html
 
-COPY nginx/ /etc/nginx/
```

Comment:
Can you please revert these changes? I believe there was a rebase/merging issue with this branch.

Reply:
Done